pyproject.toml

```diff
@@ -38,7 +42,6 @@ pytest = "*"
 pyyaml = "*"
 pydocstyle = "*"
 m2r2 = "*"
-sphinx = "*"
 sphinx-rtd-theme = "*"
```

Comment:
can likely remove as well, (missed by me)

Reply:
along with m2r2

Reply:
removing m2r2 is good, once I remove sphinx-rtd-theme I can no longer build.

Reply:
see initial pipeline failure

Reply:
removed m2r2, left sphinx-rtd-theme for now until we determine what is using this dependency and while builds start to fail once its removed.